src/concrete/ml/torch/compile.py

```diff
@@ -158,7 +158,12 @@ def _compile_torch_or_onnx_model(
         artifacts (DebugArtifacts): Artifacts object to fill during compilation
         show_mlir (bool): if set, the MLIR produced by the converter and which is going
             to be sent to the compiler backend is shown on the screen, e.g., for debugging or demo
-        n_bits: the number of bits for the quantization
+        n_bits: number of bits for quantization, can be a single value or
```

Comment:
better to write `n_bits (Union[int, Dict[str, int]])` for all. Also, maybe indicate that the default is 8 bits